securedrop/source_app/main.py

```diff
@@ -204,7 +204,7 @@ def submit(logged_in_source: SourceUser) -> werkzeug.Response:
             min_len = InstanceConfig.get_default().initial_message_min_len
             if (min_len > 0) and (msg and not fh) and (len(msg) < min_len):
                 flash(gettext(
-                    "Your first message must be at least {} characters long.".format(min_len)),
+                    "Your first message must be at least {} characters long.").format(min_len),
```

Comment:
testing - confirmed that this case works correctly when rebased onto the `i18n-merge` branch.